README.md

```diff
@@ -406,85 +406,84 @@ With the CodePush plugin downloaded and linked, and your app asking CodePush whe
 
 The simplest way to do this is to "CodePush-ify" your app's root component. To do so, you can choose one of the following three options:
 
-**Option 1: Use the [ES7 decorator](https://github.com/wycats/javascript-decorators) syntax:**
+**Option 1: Call `codePush` on your root component and assign the result to itself after the component declaration:**
```

Comment:
Could we simplify the phrasing here to something like: `Wrap your root component`?